### ISSUE TITLE
HTML energy 

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#6312]
I appreciate prerender yet.. I'm not bot and using mobile Chrome Android 😄, ideas?

![Screenshot_20230623_114645_Chrome.jpg](https://github.com/kodadot/nft-gallery/assets/5887929/65e94228-91de-4968-99a4-52555d4db443)

**Comment by roiLeo** [comment]
Tested on pixel and it worked for me, could you check your chrome version?

**Comment by preschian** [comment]
@yangwao type "check user agent" in google and give me the result here, will check

**Comment by yangwao** [comment]
I know some ppl know me irl think I'm bot, but if even machines considers me as bot, that's baaad.


![image](https://github.com/kodadot/nft-gallery/assets/5887929/84071455-43ad-459e-ab88-3696650136c0)


https://github.com/kodadot/nft-gallery/assets/5887929/e3aadfa6-f831-4583-92f2-f492b1ebae2a

**Comment by yangwao** [comment]
if this helps

![image](https://github.com/kodadot/nft-gallery/assets/5887929/27f5b50d-8642-4304-a1c5-0c2121ac2ea0)
![image](https://github.com/kodadot/nft-gallery/assets/5887929/d8e02b92-faa7-4254-b137-116939531a83)

**Comment by Jarsen136** [comment]
I guess it's not related to `user-agent` from user side. The direct reason is that google lead the user to another cache url instead of `kodadot.xyz`

![image](https://github.com/kodadot/nft-gallery/assets/31397967/53c0a3e9-9658-413a-b8da-68519a92257c)

https://kodadot-xyz.webpkgcache.com/doc/-/s/kodadot.xyz/ This page would lead to html render result.

**Comment by preschian** [comment]
@yangwao at the moment let's disable "Signed Exchange" feature first on cloudflare dashboard https://developers.cloudflare.com/fundamentals/speed/signed-exchanges/enable-signed-exchange/#enable-sxgs

ref:
- https://developers.google.com/search/docs/appearance/signed-exchange
- https://web.dev/signed-exchanges/

**Comment by preschian** [comment]
I don't think we need "Signed Exchange" for now. even with prerender.io, it will give a bad experience to the user. visually better, but the only user can do is navigate to the other pages. the users cant interact with actions that need javascript (connect, switch chain, search, accept cookies banner)

tested on visit https://preschian.xyz/ by using userAgent bot

**Comment by yangwao** [comment]
But it improves SEO ranking? 

yet turned it off for now
![image](https://github.com/kodadot/nft-gallery/assets/5887929/1dc89c9b-fc78-4bc5-a305-caf69348268f)
![image](https://github.com/kodadot/nft-gallery/assets/5887929/b79a8911-bd2c-4cc6-9e65-27309fca3cb2)

**Comment by preschian** [comment]
> But it improves SEO ranking?

yes

imo, we can enable this once we are moving from prerendering to SSR mode. because prerendering content is intended for the bot, not for the user. meanwhile, `SXG` will serve prerendering content to the user if we enable it now

**Comment by yangwao** [comment]
Okay seems to be working now. 

Can I now enable SXG back?

**Comment by preschian** [comment]
> Can I now enable SXG back?

nope, it will error again

blocked. in favor for #2690
